docs/guides/permissions/overview.mdx

```diff
@@ -0,0 +1,214 @@
+---
+id: overview
+title: Overview
+---
+
+The Ory permission service [Keto](https://github.com/ory/keto) is based on the Google Zanzibar paper. It might differ a bit from
+other authorization services you know, so lets first clarify a few concepts.
+
+:::info Ory Cloud Availability
+
+The permission service is available in Ory Cloud. The documentation, integrations and UI are currently being worked on,
+but the API is production ready. Every existing and new project can use the service without extra fees or setup.
+
+:::
+
+## Relations and Relation Tuples
+
+The underlying data model are so-called [relation tuples](/keto/concepts/relation-tuples.mdx). They encode relations between [subjects](/keto/concepts/subjects.mdx)
+and [objects](/keto/concepts/objects.mdx). Please head over to the individual pages for
+more details.
+
+Examples for relation tuples are:
+
+- `user1` is `member` of `groups:group1`
+- `member` of `groups:group1` is `reader` of `files:file1`
+
+As you can see, the subject of a relation tuple can either be a specific subject ID, or subjects defined through an indirection
+(all members of a certain group).
+
+## Checking Permissions
+
+Permissions are just another form of relations. Therefore, a permission check is a request to check whether a subject has a
+certain relation to an object, possibly through one or more indirections.
+
+Following the example above, one could ask:
```

Comment:
> the example above
This orientation-based reference isn't clear. It's better to recall the example again to avoid any misunderstandings.